index.html

```diff
@@ -32,9 +48,23 @@ <h2>Add new books</h2>
         <br />
         <br />
         <button class="button" type="submit" id="btn">Add</button>
+        <br />
+        <br />
+        <span></span>
       </form>
-      <br />
-      <span></span>
+    </div>
+
+    <div class="contact-info">
+      <h2 class="head">Contact Information</h2>
+      <p class="contact-details">Do you have any questions or just want to say "Hello"?
+        <br>
+        You can reach out to us!
+      </p>
+      <ul>
+        <li>Our email: mail@mail.com</li>
+        <li>Our phone number: 0025469875425</li>
+        <li>Our address: First St SE in Washington, DC 20004</li>
+      </ul>
     </div>
```

Comment:
- [ ] I will also suggest that you center the contact information properly and ensure that it takes up enough space on the page as well just as seen in the given wireframe :wink: 

| Your Design | Given Wireframe |
|--------|--------|
| ![image](https://user-images.githubusercontent.com/102699958/234582916-da048501-7f3e-4619-a98c-a6c3abe29320.png) | ![image](https://user-images.githubusercontent.com/102699958/234583121-d0c588e8-6167-4d9a-a257-6d34583697fb.png) |

Reply:
well resolved! ✔️